cgp/genome.py

```diff
@@ -361,6 +361,14 @@ def mutate(self, mutation_rate: float, rng: np.random.RandomState):
         bool
             True if only inactive regions of the genome were mutated, False otherwise.
         """
+
+        def count_dna_differences(dna_0: List[int], dna_1: List[int]) -> int:
+            dna_diff = 0
+            for gene_0, gene_1 in zip(dna_0, dna_1):
+                if gene_0 != gene_1:
+                    dna_diff += 1
+            return dna_diff
```

Comment:
This can be written as a one-liner:

Reply:
```suggestion
        def count_dna_differences(dna0: List[int], dna1: List[int]) -> int:
            return len([1 for gene0, gene1 in zip(dna0, dna1) if gene0 != gene1])
```

Reply:
This is just a personal preference (I think), but I'd prefer appending numbers without a `_` to variable names, i.e. to write `dna0, dna1, gene0, gene1` in this case.

Reply:
regarding the one liner: please show both to your colleague for 5sec and ask them if the understand what's happening ;) i do like compact code but it feels like this change would be sacrificing readability

Reply:
Hmm, I understand your point in general, but in this case I think the list comprehension is pretty readable and more readable than the original code.

Reply:
ok